TwitchClient/Game.swift

```diff
@@ -1,35 +0,0 @@
-//
-//  Game.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Game: NSObject {
-
-    var name : String?
-    var popularity : NSInteger?
```

Comment:
Looking at the variable name and type I have no idea what it does. How can popularity be an int?
Speaking of which, `NSInteger` is Objective-C, use `int`.